contrib/kind-common

```diff
@@ -320,7 +320,23 @@ is_nested_virt_enabled() {
 }
 
 install_kubevirt() {
-    local kubevirt_version="$(curl -L https://storage.googleapis.com/kubevirt-prow/release/kubevirt/kubevirt/stable.txt)"
+    # possible values:
+    # stable - install newest stable (default)
+    # vX.Y.Z - install specific stable (i.e v1.3.1)
+    # nightly - install newest nightly
+    # nightly tag - install specific nightly (i.e 20240910)
```

Comment:
Can you add a link in the comment to where one could look for specific nightlies?

Reply:
only thing i know is taking the date part from https://quay.io/repository/kubevirt/virt-handler?tab=tags&tag=latest

i prefer to not comment it in the file itself, it aint clean

Reply:
yeah, if they don't have a proper API for it, then not needed..
I was thinking they may have something like our cnv-explorer or something..
so all good